### PR TITLE
feat: Update generator-ibm-cloud-enablement to 1.9.4

### PR DESCRIPTION
**Pull request by djones6**
Updates `generator-ibm-cloud-enablement` to `1.7.40` to pick up Dockerfile updates (ibm-developer/generator-ibm-cloud-enablement#564).

This was originally part of #708 but there is an outstanding issue regarding a change to the quoting around the `command` parameter in `manifest.yml` that is generated by that repo, which is being rejected by the tests.

### CODE REVIEW

**Comment by gee4vee** [comment]
@djones6 I fixed the unit test to account for the removed quotes. We don't think they are necessary. Let me know if you think otherwise.

Also, by this time the cloud enablement generator is at v1.9.0. If you like you can modify the package json to go to that version. I'll leave it up to you and approve this after.

**Comment by gee4vee** [comment]
For the record this is the change when we removed the surrounding double quotes in the cloud enablement generator:
https://github.com/ibm-developer/generator-ibm-cloud-enablement/commit/93cf82cd2ff4548279123db1aa6f91e8b1612af6

**Review by nfstein — APPROVED** [review]

**Comment by djones6** [comment]
@gee4vee thanks for fixing the test.  Looks like the same fix is needed to one of the integration tests, I just pushed a fix for it.

I can see there have been a flurry of releases of generator-ibm-cloud-enablement recently.  I think it's safer to leave this part up to you, as you probably understand what is happening in that project better than I.  Please go ahead and update my branch if you think we should move to the latest version.

**Pull request merged**
